app/routes/index.tsx

```diff
@@ -126,6 +126,9 @@ export default function App() {
           />
         ))}
       </main>
+      <a id="discordChatBtn" href="https://discord.com/invite/7wjJbFJnSN">
```

Comment:
is it on purpose to use a different link that the one in footer few lines below, @plexish ?

Reply:
this one is an invite, while the other one just opens it, from what I understand

Reply:
the `https://discord.gg/vjFSCDyMCy` is also an invite, it redirects to `https://discord.com/invite/vjFSCDyMCy` (e.g. using https://unshorten.me)

Reply:
hm, yes, good point, they do go to different channels. I think ideally these people should be sent to #editing, which is actually https://discord.com/invite/Bt8PaRTDQC

Reply:
Updated URL